src/main/java/nextstep/step2/domain/ParticipantName.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.step2.domain;
+
+public class ParticipantName {
+
+    private final int NAME_MAX_RANGE = 5;
+    private final String name;
```

Comment:
```suggestion
    private final int NAME_MAX_RANGE = 5;

    private final String name;
```

상수와 인스턴스 변수 사이에는 한 줄 띄워 주시면 좋아요.